### PR TITLE
Clean Up

### PR DESCRIPTION
**Pull request by temp1011**
Mainly removing decompile artifacts. It looks like there are a lot of unused trees in the world gen section that might need cleaning up. There also seems to be a lot of reused code but I'm not sure of a nice way to reduce it.

### CODE REVIEW

File: extrabees/src/main/java/binnie/extrabees/alveary/AlvearyLogicFrameHousing.java [hunk 1]
```diff
@@ -75,36 +74,44 @@ public void wearOutEquipment(final int amount) {
 
 	@Nullable
 	private IHiveFrame getHiveFrame() {
+		if (getFrameStack() != null) {
+			return (IHiveFrame) getFrameStack().getItem();
```
Comment (mezz) [inline extrabees/src/main/java/binnie/extrabees/alveary/AlvearyLogicFrameHousing.java]: For situations like this, I like to store the result of `getFrameStack()` to a value so it is not called twice.

- Slightly better performance, any method call like `getFrameStack()` may do some calculation.
- Static analysis will work when it's trying to figure out if the `.getItem()` call is null-safe. As far as the analyzer knows, the result of `getFrameStack()` might be different each time it's called.

File: extrabees/src/main/java/binnie/extrabees/alveary/AlvearyLogicFrameHousing.java [hunk 2]
```diff
@@ -75,36 +74,44 @@ public void wearOutEquipment(final int amount) {
...
+		}
+		return null;
+	}
+
+	@Nullable
+	private ItemStack getFrameStack() {
```
Comment (mezz) [inline extrabees/src/main/java/binnie/extrabees/alveary/AlvearyLogicFrameHousing.java]: The preferred way of handling `ItemStack` in 1.12 and higher is never have them be `@Nullable`.
Most of the time you might use `null` to signify a missing `ItemStack`, it's better to use an empty one instead.
That way we are not checking for `!= null` and `!itemStack.isEmpty()` every time we use an `ItemStack`.

**Review by mezz — CHANGES_REQUESTED** [review]
Looks pretty good, just some minor requests.

**Comment by temp1011** [comment]
Hopefully this sorts this out. Sorry for slow response.

**Review by mezz — APPROVED** [review]
Thanks, looks good!

**Pull request merged**